### PR TITLE
feat: adding the ability to specify a request header

### PR DESCRIPTION
**Pull request by erunion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,19 +51,20 @@ fetchHar(har)
...
 
 - `har` is a [har](https://en.wikipedia.org/wiki/.har) file format.
+- `userAgent` is an optional user agent string to let you declare where the request is coming from.
 
-Performs a fetch request from a given HAR file. HAR files can be used to list lots of requests
-but we only use the first from the `log.entries` array.
+Performs a fetch request from a given HAR file. HAR files can be used to list lots of requests but we only use the first from the `log.entries` array.
 
-### `fetchHar.constructRequest(har) => Request`
+### `fetchHar.constructRequest(har, userAgebt) => Request`
```
Comment (dok) [inline README.md]:
```suggestion
### `fetchHar.constructRequest(har, userAgent) => Request`
```

Comment (erunion) [reply README.md]: oops

File: index.js [hunk 2]
```diff
@@ -26,11 +27,15 @@ function constructRequest(har) {
...
+  if (userAgent) {
+    options.headers['User-Agent'] = userAgent;
+  }
+
   return new Request(`${url}${querystring}`, options);
 }
 
-function fetchHar(har) {
-  return fetch(constructRequest(har));
+function fetchHar(har, userAgent) {
```
Comment (dok) [inline index.js]: Wouldn't the HAR have the user agent information in there already? Maybe I am misunderstanding the intent here

Comment (erunion) [reply index.js]: Since we don't want the "ReadMe API Explorer" user agent to show up in the code samples, we shouldn't modify the HAR.

Comment (dok) [reply index.js]: Got it.

Comment (dok) [reply index.js]: Got it.

**Pull request merged**